packages/carousel/src/styledComponents/Carousel.js

```diff
@@ -19,32 +31,28 @@ const Container = styled.div`
     display: inline-flex;
     align-items: center;
 
+    transform: translateX(
+      ${props => -(props.activeItem * stepDistance(props))}px
+    );
+    transition: transform ${props => props.itemConfig.transitionTime}s;
+    will-change: transform;
+
     li {
       margin: ${props => props.itemConfig.margin}px;
-      transition: all ${props => props.itemConfig.transitionTime}s;
       width: ${props => props.itemConfig.width}px;
       height: ${props => props.itemConfig.height}px;
 
+      transition: all ${props => props.itemConfig.transitionTime}s;
```

Comment:
Maybe we'll get a little performance improvement if we set the transition only for the objects that we change (`transform` and `margin`)

Reply:
Good catch! It was changing opacity too. I'll change it